src/aleph_client/commands/utils.py

```diff
@@ -33,6 +34,20 @@ def colorful_json(obj: str):
     )
 
 
+
+
+def colorize_status(status: MessageStatus) -> str:
```

Comment:
The result has been processed, so I suggest the past form, else I would expect the function to return a callable.
```suggestion
def colorized_status(status: MessageStatus) -> str:
```

Reply:
Why no test on this function ?